src/NuGet.Core/NuGet.Commands/Strings.resx

```diff
@@ -1092,4 +1092,10 @@ Non-HTTPS access will be removed in a future version. Consider migrating to 'HTT
     <value>NuGetAudit is enabled, but no package sources contain known vulnerability data.</value>
     <comment>Do not translate NuGetAudit</comment>
   </data>
+  <data name="Error_InvalidNuGetAuditValue" xml:space="preserve">
+    <value>Invalid NuGetAudit value '{0}'. Valid values: {1}</value>
+    <comment>Don't translate 'NuGetAudit'
+{0} - is the value from the customer's project file
+{1} is the list of valid values "true, false"</comment>
```

Comment:
How is this better than making `true, false` part of the resource string with a comment not to translate `true` and `false`?

Reply:
Firstly, I lack confidence that [humans actually translate our strings](https://www.reddit.com/r/CasualUK/comments/12cwylk/microsoft_has_mistranslated_zip_files_as_postcode/?rdt=48612). We also have [evidence that the localization team doesn't see our comments](https://github.com/NuGet/NuGet.Build.Localization/pulls?q=is%3Apr+lock+is%3Aclosed). Apparently we have to do some stuff in the loc repo, and looking at those PR diffs, I can't figure out what I would need to do to lock the `true, false` strings correctly.

So, this way is better because 1. it doesn't require us to figure out [the bizzare syntax and tooling used by our loc repo](https://github.com/NuGet/NuGet.Build.Localization/pull/613/files). 2. It reduces (eliminates) risk that we do it wrong, and despite out best efforts, French customers get told to set `<NuGetAudit>faux</NuGetAudit>`.

Reply:
We'll update in another PR if we can figure out the locking syntax. It's really unfortunate that the tooling is so difficult.